### PR TITLE
Added title

### PR DESCRIPTION
**Pull request by koldovsky**

### CODE REVIEW

File: EneidaFULL.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿Еней був парубок моторний
```
Comment (Marian3912) [inline EneidaFULL.txt]: why this line was removed?

File: EneidaFULL.txt [hunk 2]
```diff
@@ -1,4 +1,5 @@
...
+﻿- ЕНЕЇДА -
+﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿﻿Еней був парубок моторний
```
Comment (Marian3912) [inline EneidaFULL.txt]: why this line was added?

File: EneidaFULL.txt (hunk 1 above)
Comment (koldovsky) [reply EneidaFULL.txt]: Because ...

File: EneidaFULL.txt (hunk 2 above)
Comment (koldovsky) [reply EneidaFULL.txt]: ???

**Pull request merged**